Makefile

```diff
@@ -54,6 +54,7 @@ conda-lock.yml: pyproject.toml
 .PHONY: pudl-dev
 pudl-dev:
 	${mamba} run --name base ${mamba} env remove --name pudl-dev
+	${mamba} run --name base ${mamba} install --yes conda-lock
```

Comment:
I think this should address the "no `conda-lock`" issue that you ran into @rousik